bin/validate-fixture-version.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+main(){
+    export TERMINUS_HIDE_GIT_MODE_WARNING=1
+    local DIRNAME=$(dirname "$0")
+
+    if [ -z "${TERMINUS_SITE}" ]; then
+        echo "TERMINUS_SITE environment variable must be set"
+        exit 1
+    fi
+
+    if ! terminus whoami > /dev/null; then
+        if [ -z "${TERMINUS_TOKEN}" ]; then
+            echo "TERMINUS_TOKEN environment variable must be set or terminus already logged in."
+            exit 1
+        fi
+        terminus auth:login --machine-token="${TERMINUS_TOKEN}"
+    fi
+
+
```

Comment:
Nit: Same line break as https://github.com/pantheon-systems/wp-saml-auth/pull/354#discussion_r1291674421

Reply:
next time just suggest and commit things like this 😆